src/platform_impl/web/web_sys/mod.rs

```diff
@@ -171,6 +171,11 @@ pub fn set_canvas_style_property(raw: &HtmlCanvasElement, property: &str, value:
         .unwrap_or_else(|err| panic!("error: {err:?}\nFailed to set {property}"))
 }
 
+pub fn get_canvas_style_property(raw: &HtmlCanvasElement, property: &str) -> Option<String> {
+    let style = raw.style();
+    style.get_property_value(property).ok()
+}
```

Comment:
We don't do this anymore because we cache the `CssStyleDeclaration`.

Reply:
Yeah removed the whole thing because it was used for hidden detection and we decided to not fix that in this pr.